oap-server/server-query-plugin/status-query-plugin/src/main/java/org/apache/skywalking/oap/query/debug/DebuggingHTTPHandler.java

```diff
@@ -109,8 +111,12 @@ public DebuggingHTTPHandler(final ModuleManager manager, final StatusQueryConfig
     }
 
     @Get("/debugging/config/dump")
-    public String dumpConfigurations() {
-        return serverStatusService.dumpBootingConfigurations(config.getKeywords4MaskingSecretsOfConfig());
+    public String dumpConfigurations(HttpRequest request) {
+        if ("application/json".equalsIgnoreCase(request.headers().get(HttpHeaderNames.ACCEPT))) {
```

Comment:
This is fragile, some tools/browsers/sdks might send Accept header like this `Accept: application/json; charset=utf-8`

Reply:
Updated here and similar codes in TTL status APIs